pages/compiling/installation-reqs.rst

```diff
@@ -113,12 +113,21 @@ Conda Requirements
 * After you've :ref:`cloned the source code <sec-source-code>`, configure the Cantera build by
   adding the following options to a file called ``cantera.conf`` in the root of the source directory
   (creating the file if it doesn't exist):
+  
```

Comment:
Can you please delete the colon on the line above this one?

Reply:
Done.